src/rundialog/LauncherButton.vala

```diff
@@ -0,0 +1,60 @@
+/**
+ * Simple widget that shows an application icon, name, and description.
+ */
+public class LauncherButton : Gtk.Box {
+	public Budgie.Application application { get; construct; }
+
+	construct {
+		this.get_style_context().add_class("launcher-button");
+
+		var image = new Gtk.Image.from_gicon(application.icon, Gtk.IconSize.DIALOG) {
+			pixel_size = 48,
+			margin_start = 8
+		};
+
+		var right_box = new Gtk.Box(Gtk.Orientation.VERTICAL, 6);
+
+		var app_name = Markup.escape_text(application.name);
+		var sdesc = application.description;
+		if (sdesc == null) {
+			sdesc = "";
+		}
```

Comment:
What about: 

```
var sdesc = application.description ?? "";
```

Reply:
What even is that syntax lol. TIL

Reply:
It's a nullish coalescence :) It will use any truthy value on the left and fallback to the value on the right if the value is falsey.